docs/issues/Issues.md

```diff
@@ -0,0 +1,37 @@
+#Issues
+
+##Labels
+All labels available: https://github.com/HeRAMS-WHO/herams-backend/labels
+
+##Creating an issue
+###Description
+When creating an issue be descriptive as possible. A good approach is:
+
+"I am logged in as `...` with `...`, and am on page `...`. I want to `...`"
+
+From this description it should be clear for developers to reproduce the taken steps to get to the problem.
+The desired next steps and features should be just as clear.
+
+###Assigning labels
+Labels must be assigned to make it very clear what the status of the issue is. Use the following issue categories:
+* **env** Indicate on which platform the error occurs, not needed when it is a new feature  
+* **urgency** When the urgency is other than normal, assign an urgency label like _extreme_, _high_ or _low_
+* **subject** Help developers to filter issues to pick up quickly by adding a subject
+* **type** Help developers prioritize issues withing an urgency, i.e. bug often is picked up before an enhancement
+
+##Issue flow
+###Assignee
+The developers will assign the issues to themselves when they start working on an issue.
+
+###Flow
+* When there is **an assignee**, it can be assumed the issue is in progress
+* When the implementation is complete and released on staging, a developer will add **flow:test on staging**
+* It must then be checked by the reporter on staging:
```

Comment:
Due to the load on @luna404 for small issues we should self test and self report. Maybe, when @luna404 is the reporter she should mark if she wants to be involved in testing the solution or not. (For minor issues it is reasonable to make it a fire & forget)